website/pages/blog/graphqxl-language.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: GraphQXL - The missing GraphQL Language Extension?
+tags: [graphql]
+authors: [gabotechs]
+date: 2022-11-30
```

Comment:
need to adjust the date before publishing. @Urigo @dotansimha lets publish it today?